images/build/__common.sh

```diff
@@ -0,0 +1,19 @@
+function downloadFileAndVerifyChecksum() {
+    local platformName="$1"
+    local version="$2"
+    local downloadedFileName="$3"
+    local headersFile="/tmp/headers.txt"
+
+    echo "Downloading $platformName version '$version'..."
+    curl \
+        -D $headersFile \
+        -SL "https://oryxsdksdev.blob.core.windows.net/$platformName/$platformName-$version.tar.gz" \
```

Comment:
nit: keep this link 'https://oryxsdksdev.blob.core.windows.net'  in constant.yaml?

Reply:
Good point